modules/mobile/spec/request/v1/user_request_spec.rb

```diff
@@ -299,7 +299,7 @@
           VCR.use_cassette('mobile/payment_information/payment_information') do
             VCR.use_cassette('mobile/user/get_facilities') do
               VCR.use_cassette('mobile/va_profile/demographics/demographics') do
-                get '/mobile/v0/user', headers: iam_headers
```

Comment:
These are not related to the PR. They were mistakes that were made when creating the v1 of this spec.